proto/streamlit/proto/Checkbox.proto

```diff
@@ -19,5 +19,5 @@ syntax = "proto3";
 message Checkbox {
   string id = 1;
   string label = 2;
-  bool default = 3;
+  bool value = 3;
```

Comment:
Like Button.proto comment: To change the protobuf, choose a new numerical value for "value" and explicitly deprecate the word "default" as well as its number.